static/src/javascripts/projects/common/modules/ui/relativedates.js

```diff
@@ -0,0 +1,234 @@
+// @flow
+import $ from 'lib/$';
+import mediator from 'lib/mediator';
+import bonzo from 'bonzo';
+
+type relativeDateOptions = {
```

Comment:
Is it acceptable to create this opts with everything optional or should I go further and try and tidy this up here?

Reply:
I think it's reasonable, unless any of them can take default values (e.g. { format: 'short' }). 

Also the convention we have been using is to UpperCamelCase type names.